json-to-go.js

```diff
@@ -321,7 +321,9 @@ function jsonToGo(json, typename, flatten = true)
 	function toProperCase(str)
 	{
 		// ensure that the SCREAMING_SNAKE_CASE is converted to snake_case
-		str=str.toLowerCase();
+    if (str.match(/^[_A-Z0-9]+$/)) {
+      str=str.toLowerCase();
+    }
```

Comment:
Please clean up the indentation and spacing here

```suggestion
		if (str.match(/^[_A-Z0-9]+$/)) {
 			str = str.toLowerCase();
		}
```

Reply:
done. thanks